docs/user-guide/interfaces/cli/using-rugs.md

```diff
@@ -26,20 +35,78 @@ Searching catalogs completed
 ...
 ```
 
-The output informs you about the name of the Rug generator and which
-Rug archive it lives in.
+When you use the `--operations` command-line argument, the output
+lists the name of the Rug and its Rug archive.  Providing the `--type
+RUG_TYPE` command-line option limits the results to Rugs of that type,
+e.g., `generator` or `editor`.
+
+## Describe
+
+To get information about a Rug and list all its parameters, run the
+`rug describe` command.
+
+```console
+$ rug describe archive atomist-rugs:spring-rest-service
+
+$ rug describe editor atomist-rugs:spring-boot-rest-service:SpringBootThing \
+    --artifact-version=1.0.0
+
+$ rug describe generator atomist-rugs:spring-boot-rest-service:NewSpringBootThing
+```
+
+Providing `--artifact-version` is optional and defaults to `latest`
+semantics.
+
+## List
+
+To list all locally available Rug archives, run the `rug list`
+command:
+
+```console
+$ rug list -f 'version=[1.2,2.0)' -f 'group=*atomist*' -f 'artifact=*sp?ing*'
+```
+
+The local listing can be filtered by using `-f` filter expressions on
+`group`, `artifact` and `version`. `group` and `artifact` support
+wildcards of `*` and `?`.  `version` takes any version constraint.
+
+## Edit
+
+Run an editor as follows:
+
+```console
+$ rug edit atomist-rugs:common-editors:AddReadme --artifact-version=1.0.0 \
+    parameter1=foo parameter2=bar
+
+$ rug edit atomist-rugs:common-editors:AddReadme parameter1=foo parameter2=bar
+```
+
+Providing `--artifact-version` is optional and defaults to `latest`
+semantics.  `--change-dir` or `-C` to tell the CLI to apply the editor
+to a directory other than the current working directory.
 
 ## Generators
 
 You can create new projects on your local machine using the CLI, for instance:
 
-```
+```console
 $ cd ~/workspace
 $ rug generate atomist-rugs:rug-project:NewStarterRugProject my-new-generator
 ```
 
-The first argument is always the project name.
+The first argument is always the project name.  If the generator
+requires additional parameters, just append the `parameter=value` to
+the end of the command line.
+
+```console
+$ rug generate atomist-rugs:spring-boot-rest-service:NewSpringBootRestService" \
+    --artifact-version=1.0.0 my-new-project parameter1=foo parameter2=bar
+
+$ rug generate atomist-rugs:spring-boot-rest-service:NewSpringBootRestService" \
+    my-new-project parameter1=foo parameter2=bar
+```
 
```

Comment:
It might make sense to add a quick note about `--interactive` for both editors and generators as I think it makes using them a lot easier.